src/main/kotlin/no/skatteetaten/aurora/gobo/graphql/application/Application.kt

```diff
@@ -29,7 +30,10 @@ data class Application(
             .firstOrNull { it.registryUrl != null && !DockerRegistryUtil.isInternal(it.registryUrl) }
 }
 
-data class ApplicationEdge(val node: Application) : GoboEdge(node.name) {
+data class ApplicationEdge(
+    @Deprecated(message = "edges.node is deprecated", replaceWith = ReplaceWith("items"))
```

Comment:
Denne vil komme i error-bolken. Blir det komplikasjoner for konsollet ? Ref det vi snakket med Martin om.

Reply:
Denne vil ikke komme i error blokken, for her funker det å bruke @Deprecated.
En advarsel vil vises i feks playground og i graphql skjemaet, hvis man bruker edges

Reply:
OK!